### PR TITLE
Fix REMOTETAGS

### PR DESCRIPTION
**Pull request by rhatdan**
Handle REMOTETAGS the same way for all remote commands.

This fixes issues where remote commands are not building correctly on rhel7 and centos7 systems.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,7 @@ ETCDIR ?= /etc
 TMPFILESDIR ?= ${PREFIX}/lib/tmpfiles.d
 SYSTEMDDIR ?= ${PREFIX}/lib/systemd/system
 USERSYSTEMDDIR ?= ${PREFIX}/lib/systemd/user
+REMOTETAGS := !ABISupport remoteclient exclude_graphdriver_btrfs btrfs_noversion  exclude_graphdriver_devicemapper containers_image_openpgp
```
Comment (mheon) [inline Makefile:25]: Should we use the hack/ scripts to determine whether btrfs, seccomp, etc are included?

Comment (rhatdan) [reply Makefile:25]: No, We don't want this code compiled into remoteclient. In a perfect world we would not hit this.  The btrfs and devicemapper flags are for containers/image and containers/storage and prevent btrfs and devicemapper from being compiled into the client.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by mheon** [comment]
Manually setting hold and LGTM

**Pull request merged**